types/glider-js/index.d.ts

```diff
@@ -184,85 +186,97 @@ declare namespace Glider {
-        opt: Options;
-        page: number;
-        preventClick: boolean;
-        slide: number;
-        slides: HTMLCollection;
-        track: HTMLElement;
-        trackWidth: number;
-
-        // tslint:disable-next-line no-misused-new
-        new <T extends HTMLElement = HTMLDivElement>(ref: T, options: Options): Static<T>;
-
-        /**
-         * Add an item to the list
-         * @param element HTMLElement
-         */
-        addItem(element: HTMLElement): void;
-
-        /**
-         * Destroy Glider.js.
-         */
-        destroy(): void;
-
-        /**
-         * Force a refresh of Glider.js. If rebuildPaging is true, paging
-         * controls will force a rebuild as well. This arument must be true
-         * if any options affecting paging controls (dots/arrows) are
-         * modified.
-         * @param rebuildPaging boolean
-         */
-        refresh(rebuildPaging: boolean): void;
-
-        /**
-         * Remove an item from the list
-         * @param itemIndex number
-         */
-        removeItem(itemIndex: number): void;
-
-        /**
-         * Scroll to any slide or page. If second argument is explicitly
-         * true, then the first argument will refer to the page to scroll
-         * to, as opposed to the slide.
-         * @param slideIndexs string | number
-         * @param isActuallyDotIndex boolean
-         */
-        scrollItem(slideIndex: string | number, isActuallyDotIndex: boolean): void;
-
-        /**
-         * Scroll directly to supplied scroll position in pixels
-         * @param pixelOffset number
-         */
-        scrollTo(pixelOffset: number): void;
-
-        /**
-         * Overrides options set during initialization. If called when a
-         * breakpoint is active, the settings for the active breakpoint are
-         * updated. If second argument is true, the global settings are
-         * updated regardless of active breakpoint. Required for overriding
-         * `responsive` setting itself
-         * @param arguments Glider.Options
-         * @param global Boolean
-         */
-        setOption(arguments: Options, global?: boolean): void;
-
-        /**
-         * Force a refresh of Glider.js navigation controls
-         */
-        updateControls(): void;
+    interface Arrow extends HTMLElement {
+        _func?: (glider: Glider, direction: 'next' | 'prev') => false;
     }
 }
 
+declare class Glider<T extends HTMLElement = HTMLDivElement> {
+    animate_id: number;
+    arrows: {
+        next: Glider.Arrow;
+        prev: Glider.Arrow;
+    };
+    breakpoint: number;
+    containerWidth: number;
+    dots: HTMLElement;
+    ele: T;
+    isDrag: boolean;
+    itemWidth: number;
+    opt: Glider.Options;
+    page: number;
+    preventClick: boolean;
+    slide: number;
+    slides: HTMLCollection;
+    track: HTMLElement;
+    trackWidth: number;
+
+    private _opt: Glider.Options;
+
+    constructor(ref: T, options: Glider.Options);
```

Comment:
```suggestion
    constructor(ref: T, options?: Glider.Options);
```
```js
new Glider(document.querySelector('.glider'));
```
(from readme, works via `Object.assign`